apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/LoadRdaJsonApp.java

```diff
@@ -0,0 +1,159 @@
+ * data so either or both can be provided as needed.
+ */
+public class LoadRdaJsonApp {
+  private static final Logger logger = LoggerFactory.getLogger(LoadRdaJsonApp.class);
+
+  public static void main(String[] args) throws Exception {
+    final ConfigLoader.Builder options = ConfigLoader.builder();
+    if (args.length == 1) {
+      options.addPropertiesFile(new File(args[0]));
+    } else if (System.getProperty("config.properties", "").length() > 0) {
+      options.addPropertiesFile(new File(System.getProperty("config.properties")));
+    }
+    options.addSystemProperties();
+    final Config config = new Config(options.build());
+
+    final MetricRegistry metrics = new MetricRegistry();
+    final Slf4jReporter reporter =
+        Slf4jReporter.forRegistry(metrics)
+            .outputTo(LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME))
+            .convertRatesTo(TimeUnit.SECONDS)
+            .convertDurationsTo(TimeUnit.MILLISECONDS)
+            .build();
+    reporter.start(5, TimeUnit.SECONDS);
+    try {
+      logger.info("starting RDA API local server");
+      RdaServer.runWithLocalServer(
+          config::createFissClaimsSource,
+          config::createMcsClaimsSource,
+          port -> {
+            final RdaLoadOptions jobConfig = config.createRdaLoadOptions(port);
+            final DatabaseOptions databaseConfig = config.createDatabaseOptions();
+            final HikariDataSource pooledDataSource =
+                PipelineApplicationState.createPooledDataSource(databaseConfig, metrics);
+            if (config.runSchemaMigration) {
+              logger.info("running database migration");
+              DatabaseSchemaManager.createOrUpdateSchema(pooledDataSource);
+            }
+            try (PipelineApplicationState appState =
+                new PipelineApplicationState(
+                    metrics,
+                    pooledDataSource,
+                    PipelineApplicationState.RDA_PERSISTENCE_UNIT_NAME,
+                    Clock.systemUTC())) {
+              final List<PipelineJob<?>> jobs = config.createPipelineJobs(jobConfig, appState);
+              for (PipelineJob<?> job : jobs) {
+                logger.info("starting job {}", job.getClass().getSimpleName());
+                job.call();
+              }
+            }
+          });
+    } finally {
+      reporter.report();
+      reporter.close();
+    }
+  }
+
+  private static class Config {
+    private final String hashPepper;
+    private final int hashIterations;
+    private final String dbUrl;
+    private final String dbUser;
+    private final String dbPassword;
+    private final int batchSize;
+    private final boolean runSchemaMigration;
+    private final Optional<File> fissFile;
+    private final Optional<File> mcsFile;
+
+    private Config(ConfigLoader options) {
+      hashPepper = options.stringValue("hash.pepper", "notarealpepper");
+      hashIterations = options.intValue("hash.iterations", 2);
```

Comment:
any reason for a default of 2 iterations here compared to the one specified in `DirectRdaLoadApp.readRdaLoadOptionsFromProperties` (which is 100)?

Reply:
Since the default pepper isn't real I didn't think it mattered if the iterations were real either and 2 is cheaper than 100.  It could be changed though.  When loading real data we'd use a configuration that made these explicit and the default wouldn't matter.